### PR TITLE
Send dialog

### PR DESCRIPTION
**Pull request by tomsmalley**

### CODE REVIEW

File: frontend/src/Frontend/UI/Dialogs/Send.hs [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        & inputElementConfig_initialValue .~ unAccountName (_account_name gasPayer)
+      mkLabeledInput True "Recipient Address" uiInputElement $ def
+        & initialAttributes .~ "disabled" =: "disabled"
+        & inputElementConfig_initialValue .~ textKadenaAddress recipient
+    elClass "h2" "heading heading_type_h2" $ text "Transaction Details"
+    divClass "group" $ do
+      mkLabeledInput True "Amount" (dimensionalInputWrapper "KDA" . uiInputElement) $ def
+        & initialAttributes .~ "disabled" =: "disabled"
+        & inputElementConfig_initialValue .~ tshow amount
+      -- The designs show gas fees here, but we can't get that information yet.
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/Dialogs/Send.hs]: TODO for grepability?

File: frontend/src/Frontend/UI/Dialogs/Send.hs [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  -> KadenaAddress -- ^ To
+  -> Decimal -- ^ Amount
+  -> ([Either Text NodeInfo], PublicMeta, NetworkName) -- ^ Misc network information
+  -> Workflow t m (mConf, Event t ())
+sendDeploy _model sender gasPayer recipient amount (nodeInfos, publicMeta, networkId) = Workflow $ do
+  let code = T.unwords
+        [ "(coin.transfer"
+        , T.pack . show $ unAccountName $ _account_name sender
+        , T.pack . show $ unAccountName $ _kadenaAddress_accountName recipient
+        , tshow amount
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/Dialogs/Send.hs:125]: o_O?
2x `T.pack . show`
1x `tshow`

**Review by benkolera — APPROVED** [review]
LGTM, but don't trust the beginner eyes. :)

File: frontend/src/Frontend/UI/Dialogs/Send.hs (hunk 2 above)
Comment (tomsmalley) [reply frontend/src/Frontend/UI/Dialogs/Send.hs:125]: I like to mix things up a little.

**Pull request merged**